src/app/modules/item/pages/item-edit/item-edit.component.ts

```diff
@@ -368,4 +370,16 @@ export class ItemEditComponent implements OnDestroy, PendingChangesComponent {
     this.itemForm.enable();
     this.editContent?.reset();
   }
+
+  private maxTeamSizeValidator(): ValidatorFn {
+    return (itemForm): null => {
+      const isParticipationAsTeamOnly = itemForm.get('entry_participant_type')?.value as boolean;
+      const maxTeamSizeControl = itemForm.get('entry_max_team_size');
+      if (!maxTeamSizeControl) return null;
+
+      const errors = isParticipationAsTeamOnly ? Validators.min(1)(maxTeamSizeControl) : null;
+      maxTeamSizeControl.setErrors(errors);
+      return null;
+    };
+  }
```

Comment:
I would have preferred a fieldset level validator too, it would look like this. If it's ok with you, we can leave it as is.

However, the trouble with this approach is that I need to manually attach the error to the max team size control instead of returning the validation errors (which would attach them to the form group control), and that is a side effect to me.
And the reason why I need to attach the error to the max_team_size control is because it's currently the only/preferred way to monitor errors on the ui.